### ISSUE TITLE
upgrade/migrate for Plone 5: isaw.policy 1.0.dev0

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#523]

**Comment by paregorios** [comment]
Data will be migrated fine, but the views need to be checked in Plone 5.1.

**Comment by sauzher** [comment]
@paregorios  
`isaw.policy.location` is still needed even if there is no active contents in the portal?

the ticket you refer to has been created?

**Comment by paregorios** [comment]
@sauzher Thanks. `isaw.policy.location` is no longer needed and can be eliminated. I have not created a new ticket for the problems, and think we don't need one since we are going to eliminate the type entirely. Does that make sense to you?

**Comment by sauzher** [comment]
Yes, it does. Now I can get rid of that type during migration script run.
Thanks.

**Comment by paregorios** [comment]
`isaw.policy.location` still appears in the `portal_types` tool in the ZMI. Apparently is was not removed during the migration script run [as mentioned above](https://github.com/isawnyu/isaw.web/issues/523#issuecomment-2541781273). 

@sauzher what is the status of this work?

**Comment by paregorios** [comment]
Tom to check that this is fixed on 5x staging now

**Comment by skleinfeldt** [comment]
@sauzher says `isaw.policy.location` should not appear in the `portal_types` tool any more.
